Services/Administration/OJS.Services.Administration.Business/ProblemResources/ProblemResourceBusinessService.cs

```diff
@@ -47,13 +48,38 @@ public override async Task<ProblemResourceAdministrationModel> Edit(ProblemResou
             .Include(pr => pr.Problem)
             .FirstOrDefaultAsync();
 
+        var areFileAndLinkNull = model is { File: null, Link: null };
+
+        if (resource is { File: null } && areFileAndLinkNull)
+        {
```

Comment:
The `resource is null` check should come before this one IMO

Reply:
Done.